### PR TITLE
take-aws-cni-version-from-release

### PR DESCRIPTION
**Pull request by calvix**
remove static aws cni version and use the one we define in the release

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ workflows:
       - architect/go-build:
           name: go-build
           binary: aws-operator
-          resource_class: large
+          resource_class: xlarge
```
Comment (calvix) [inline .circleci/config.yml:25]: I have seen many errors building aws-operator  since the process was killed due low resources so bumping.

**Comment by calvix** [comment]
tested on ginger, it works

File: service/internal/images/images.go [hunk 2]
```diff
@@ -117,6 +117,26 @@ func (i *Images) Versions(ctx context.Context, obj interface{}) (k8scloudconfig.
 	return v, nil
 }
 
+func (i *Images) AWSCNI(ctx context.Context, obj interface{}) (string, error) {
```
Comment (xh3b4sd) [inline service/internal/images/images.go:120]: Can we please order the methods alphabetically? Above is `Version`. This here is `AWSCNI`.

File: service/internal/cloudconfig/template/aws-cni.go [hunk 3]
```diff
@@ -101,7 +101,7 @@ spec:
       tolerations:
         - operator: Exists
       containers:
-        - image: {{.RegistryDomain}}/giantswarm/amazon-k8s-cni:v1.6.0
+        - image: {{.RegistryDomain}}/giantswarm/amazon-k8s-cni:v{{.AWSCNIVersion}}
```
Comment (xh3b4sd) [inline service/internal/cloudconfig/template/aws-cni.go:104]: Can we safely assume the `v` prefix madness will not hunt us down at some point? I am really not certain how we define it in all the different places.

Comment (calvix) [reply service/internal/cloudconfig/template/aws-cni.go:104]: all component versions have just the sematic numbers, so having one with `v` in release would look weird

File: service/internal/images/images.go (hunk 2 above)
Comment (calvix) [reply service/internal/images/images.go]: done

**Pull request merged**